libs/langgraph/langgraph/pregel/runner.py

```diff
@@ -331,9 +331,9 @@ def writer(
                     continue
                 # schedule the next task, if the callback returns one
                 wcall = calls[idx] if calls is not None else None
-                cnt = scratchpad.call_counter
-                scratchpad.call_counter += 1
-                if next_task := self.schedule_task(task, cnt, wcall):
+                if next_task := self.schedule_task(
+                    task, scratchpad.call_counter(), wcall
```

Comment:
nit: make call_counter a property?

Reply:
ah ignore me - I misread it -it's get and increment, so there's a side effect

Reply:
it is a property, whose value is a function that returns the next value. I think auto incrementing on property getter might be a little too hard to read?

Reply:
Yeah, I came back and realized the sytanx I suggested was poor because I realized I hadn't seen where you were incrementing the counter. Revised nit might be that naming it something like `get_and_inc_call_counter()` would be more readable (albeit verbose).